src/Elliptic/Systems/Xcts/Tags.hpp

```diff
@@ -121,5 +125,112 @@ struct ShiftStrain : db::SimpleTag {
   using type = tnsr::ii<DataType, Dim, Frame>;
 };
 
+/*!
+ * \brief The conformal longitudinal operator applied to the shift excess
+ * \f$(\bar{L}\beta^\mathrm{excess})^{ij}\f$
+ *
+ * This quantity can be used as the "flux" for the momentum constraint in
+ * formulations of the XCTS equations, because the principal part of the
+ * momentum constraint is essentially the divergence of this quantity.
+ */
+template <typename DataType, size_t Dim, typename Frame>
+struct LongitudinalShiftExcess : db::SimpleTag {
+  using type = tnsr::II<DataType, Dim, Frame>;
+};
+
+/*!
+ * \brief The conformal longitudinal operator applied to the background shift
+ * vector minus the time derivative of the conformal metric
+ * \f$(\bar{L}\beta^\mathrm{background})^{ij} - \bar{u}^ij\f$
+ *
+ * Note that the time derivative of the conformal metric has its indices raised
+ * as \f$\bar{u}^ij = \bar{\gamma}^{ik}\bar{\gamma}^{jl}
+ * \partial_t\bar{\gamma}_{kl}\f$, i.e. by the conformal metric, as usual for
+ * conformal quantities.
```

Comment:
```suggestion
 * Note: as usual for conformal quantities, the indices here are raised with the conformal metric,
 * as \f$\bar{u}^ij = \bar{\gamma}^{ik}\bar{\gamma}^{jl}
 * \partial_t\bar{\gamma}_{kl}\f$.
```

(streamlined wording)